packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -344,11 +344,17 @@ function DocSidebar({
             [styles.menuWithAnnouncementBar]:
               !isAnnouncementBarClosed && showAnnouncementBar,
           },
-        )}>
+        )}
+        aria-label={translate({
+          id: 'theme.docs.sidebar.menuLabel',
+          message: 'Docs navigation',
```

Comment:
I just thought that maybe we shouldn't use the word "docs" in ARIA labels? After all, it won't always be exactly docs, right? Maybe should use "Secondary navigation" or something like that?

Reply:
Gatsby has "Secondary navigation" so why not. Isn't there value to also mention it's a sidebar? like `Secondary navigation (sidebar)` or something?

Reply:
Propose simply "Sidebar navigation".